Userland/Libraries/LibGUI/Label.cpp

```diff
@@ -131,58 +107,11 @@ void Label::size_to_fit()
     set_fixed_width(font().width(m_text));
 }
 
-void Label::wrap_text()
+int Label::preferred_height() const
 {
-    Vector<String> words;
-    Optional<size_t> start;
-    for (size_t i = 0; i < m_text.length(); i++) {
-        switch (m_text[i]) {
-        case '\n':
-        case '\r':
-        case '\t':
-        case ' ': {
-            if (start.has_value())
-                words.append(m_text.substring(start.value(), i - start.value()));
-            start.clear();
-            continue;
-        }
-        default: {
-            if (!start.has_value())
-                start = i;
-        }
-        }
-    }
-
-    if (start.has_value())
-        words.append(m_text.substring(start.value(), m_text.length() - start.value()));
-
-    auto rect = frame_inner_rect();
-    if (frame_thickness() > 0)
-        rect.set_width(rect.width() - font().glyph_width('x'));
-
-    Vector<String> lines;
-    StringBuilder builder;
-    int line_width = 0;
-    for (auto& word : words) {
-        int word_width = font().width(word);
-        if (line_width > 0)
-            word_width += font().glyph_width('x');
-        if (line_width + word_width > rect.width()) {
-            lines.append(builder.to_string());
-            builder.clear();
-            line_width = 0;
-        }
-        if (line_width > 0)
-            builder.append(' ');
-        builder.append(word);
-        line_width += word_width;
-    }
-
-    auto last_line = builder.to_string();
-    if (!last_line.is_empty())
-        lines.append(last_line);
-
-    m_lines = lines;
+    // FIXME: The 4 is taken from Gfx::Painter and should be available as
+    // a constant instead.
```

Comment:
Nit: please indent the second and later lines in these kinds of comments until after the `:`
```suggestion
    // FIXME: The 4 is taken from Gfx::Painter and should be available as
    //        a constant instead.
```

This makes (at least) CLion highlight the entire block as a FIXME and not just the first line.